src/test/bridges/liquity/StabilityPoolBridgeUnit.t.sol

```diff
@@ -61,104 +64,34 @@ contract StabilityPoolBridgeUnitTest is TestUtil {
         );
 
         // Transfer StabilityPoolBridge accounting token (SPB) back to the bridge
-        IERC20(inputAssetA.erc20Address).transfer(address(bridge), inputValue);
+        IERC20(inputAssetA.erc20Address).transfer(address(bridge), spbReceived);
+
+        // Mint rewards to the bridge
+        deal(address(bridge), _ethRewards);
+        deal(tokens["LQTY"].addr, address(bridge), _lqtyRewards);
 
         // Withdraw LUSD from StabilityPool through the bridge
+        uint256 withdrawalAmount = bound(_withdrawalAmount, 10, spbReceived);
+
         (uint256 outputValueA, , ) = bridge.convert(
             inputAssetA,
             emptyAsset,
             outputAssetA,
             emptyAsset,
-            inputValue,
+            withdrawalAmount,
             1,
             0,
             address(0)
         );
 
-        // Check the total supply of SPB token is 0
-        assertEq(bridge.totalSupply(), 0);
+        // Check the total supply of SPB token is spbReceived - withdrawalAmount
+        assertEq(bridge.totalSupply(), spbReceived - withdrawalAmount);
 
         // Transfer the funds back from the bridge to the rollup processor
-        IERC20(outputAssetA.erc20Address).transferFrom(address(bridge), rollupProcessor, outputValueA);
-
-        // Check the LUSD balance of rollupProcessor is greater or equal to the initial LUSD deposit
-        assertGe(outputValueA, inputValue);
-    }
-
-    function testMultipleDepositsWithdrawals() public {
```

Comment:
You removed this test because of the continuous swaps?

Reply:
Yes